spec/quickbooks-payments/basic_model_spec.rb

```diff
@@ -20,7 +25,18 @@ def _fakekey=(val)
   end
 
   describe 'instance methods' do
-    let(:options) { { "token" => "abcdef" } }
-    subject(:instance) { described_class.new options }
+    let(:options) { { 'token' => 'abcdef' } }
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.